core/scripts/vrfv2plus/testnet/proofs.go

```diff
@@ -0,0 +1,149 @@
+package main
+
+import (
+	"bytes"
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/shopspring/decimal"
+	helpers "github.com/smartcontractkit/chainlink/core/scripts/common"
+	"github.com/smartcontractkit/chainlink/v2/core/services/keystore/keys/vrfkey"
+	"github.com/smartcontractkit/chainlink/v2/core/services/vrf/proof"
+)
+
+var vrfProofTemplate string = `{
+	pk: [
+		%s,
+		%s
+	],
+	gamma: [
+		%s,
+		%s
+	],
+	c: %s,
+	s: %s,
+	seed: %s,
+	uWitness: %s,
+	cGammaWitness: [
+		%s,
+		%s
+	],
+	sHashWitness: [
+		%s,
+		%s
+	],
+	zInv: %s
+}
+`
+
+var rcTemplate string = `{
+	blockNum: %d,
+	subId: %d,
+	callbackGasLimit: %d,
+	numWords: %d,
+	sender: %s,
+	nativePayment: %t
+}
+`
+
+func generateProofForV2Plus(e helpers.Environment) {
```

Comment:
Seems like this is copied over from the vrfv2/testnet directory. I think you can axe it from there since it's here now.

Reply:
Removed proofs.go and support for command "generate-proof-v2-plus" from vrfv2/testnet